### ISSUE TITLE
Broken in PHP 8.0 (Named parameters)

### ISSUE THREAD

**Issue by josenobile** [elementor/elementor#13269]
Hello, 

Function tag_data_to_tag_text( $tag_id, $tag_name, array $settings = [] ) defined in https://github.com/elementor/elementor/blob/069e890bc2ee5d0d679a6ac2f3ab8a4fbabf6b9b/core/dynamic-tags/manager.php#L175

Is called in https://github.com/elementor/elementor/blob/069e890bc2ee5d0d679a6ac2f3ab8a4fbabf6b9b/core/dynamic-tags/manager.php#L123

With wrong named arguments defined in https://github.com/elementor/elementor/blob/069e890bc2ee5d0d679a6ac2f3ab8a4fbabf6b9b/core/dynamic-tags/manager.php#L144

This causes a fatal error according to PHP documentation (Introduced in PHP 8.0): https://wiki.php.net/rfc/named_params#constraints

Elementor: v3.0.14
Wordpress: 5.6
PHP: 8.0

The backtrace:
```
[09-Dec-2020 07:28:47 UTC] PHP Fatal error: Uncaught Error: Unknown named parameter $id in /home/**user**/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php:123
Stack trace:
#0 /home/**user**/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php(123): call_user_func_array()
#1 /home/**user**/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php(88): Elementor\Core\DynamicTags\Manager->parse_tag_text()
#2 [internal function]: Elementor\Core\DynamicTags\Manager->Elementor\Core\DynamicTags\{closure}()
#3 /home/**user**/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php(87): preg_replace_callback()
#4 /home/**user**/public_html/wp-content/plugins/elementor/includes/controls/base-data.php(81): Elementor\Core\DynamicTags\Manager->parse_tags_text()
#5 /home/**user**/public_html/wp-content/plugins/elementor/includes/base/controls-stack.php(1179): Elementor\Base_Data_Control->parse_tags()
#6 /home/**user**/public_html/wp-content/plugins/elementor/includes/base/controls-stack.php(1030): Elementor\Controls_Stack->parse_dynamic_settings()
#7 /home/**user**/public_html/wp-content/plugins/elementor/core/files/css/post.php(296): Elementor\Controls_Stack->get_parsed_dynamic_settings()
#8 /home/**user**/public_html/wp-content/plugins/elementor/core/dynamic-tags/dynamic-css.php(32): Elementor\Core\Files\CSS\Post->render_styles()
#9 /home/**user**/public_html/wp-content/plugins/elementor/core/dynamic-tags/dynamic-css.php(36): Elementor\Core\DynamicTags\Dynamic_CSS->render_styles()
#10 /home/**user**/public_html/wp-content/plugins/elementor/core/dynamic-tags/dynamic-css.php(36): Elementor\Core\DynamicTags\Dynamic_CSS->render_styles()
#11 /home/**user**/public_html/wp-content/plugins/elementor/core/files/css/post.php(178): Elementor\Core\DynamicTags\Dynamic_CSS->render_styles()
#12 /home/**user**/public_html/wp-content/plugins/elementor/core/files/css/base.php(617): Elementor\Core\Files\CSS\Post->render_css()
#13 /home/**user**/public_html/wp-content/plugins/elementor/core/files/base.php(166): Elementor\Core\Files\CSS\Base->parse_content()
#14 /home/**user**/public_html/wp-content/plugins/elementor/core/files/css/base.php(131): Elementor\Core\Files\Base->update_file()
#15 /home/**user**/public_html/wp-content/plugins/elementor/core/files/css/base.php(207): Elementor\Core\Files\CSS\Base->update()
#16 /home/**user**/public_html/wp-content/plugins/elementor/core/files/css/post.php(199): Elementor\Core\Files\CSS\Base->enqueue()
#17 /home/**user**/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php(438): Elementor\Core\Files\CSS\Post->enqueue()
#18 /home/**user**/public_html/wp-includes/class-wp-hook.php(287): Elementor\Core\DynamicTags\Manager->after_enqueue_post_css()
#19 /home/**user**/public_html/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters()
#20 /home/**user**/public_html/wp-includes/plugin.php(484): WP_Hook->do_action()
#21 /home/**user**/public_html/wp-content/plugins/elementor/core/files/css/base.php(254): do_action()
#22 /home/**user**/public_html/wp-content/plugins/elementor/core/files/css/post.php(199): Elementor\Core\Files\CSS\Base->enqueue()
#23 /home/**user**/public_html/wp-content/plugins/elementor/includes/frontend.php(640): Elementor\Core\Files\CSS\Post->enqueue()
#24 /home/**user**/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/locations-manager.php(90): Elementor\Frontend->enqueue_styles()
#25 /home/**user**/public_html/wp-includes/class-wp-hook.php(287): ElementorPro\Modules\ThemeBuilder\Classes\Locations_Manager->enqueue_styles()
#26 /home/**user**/public_html/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters()
#27 /home/**user**/public_html/wp-includes/plugin.php(484): WP_Hook->do_action()
#28 /home/**user**/public_html/wp-includes/script-loader.php(2004): do_action()
#29 /home/**user**/public_html/wp-includes/class-wp-hook.php(287): wp_enqueue_scripts()
#30 /home/**user**/public_html/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters()
#31 /home/**user**/public_html/wp-includes/plugin.php(484): WP_Hook->do_action()
#32 /home/**user**/public_html/wp-includes/general-template.php(3005): do_action()
#33 /home/**user**/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/views/theme-support-header.php(21): wp_head()
#34 /home/**user**/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/theme-support.php(72): require(‘/home/**user**/…’)
#35 /home/**user**/public_html/wp-includes/class-wp-hook.php(289): ElementorPro\Modules\ThemeBuilder\Classes\Theme_Support->get_header()
#36 /home/**user**/public_html/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters()
#37 /home/**user**/public_html/wp-includes/plugin.php(484): WP_Hook->do_action()
#38 /home/**user**/public_html/wp-includes/general-template.php(38): do_action()
#39 /home/**user**/public_html/wp-content/plugins/elementor/modules/page-templates/templates/header-footer.php(9): get_header()
#40 /home/**user**/public_html/wp-includes/template-loader.php(106): include(‘/home/**user**/…’)
#41 /home/**user**/public_html/wp-blog-header.php(19): require_once(‘/home/**user**/…’)
#42 /home/**user**/public_html/index.php(17): require(‘/home/**user**/…’)
#43 {main}
thrown in /home/**user**/public_html/wp-content/plugins/elementor/core/dynamic-tags/manager.php on line 123
```

**Comment by shilo-ey** [comment]
We will address this issue soon and release a patch.
Please follow this thread for updates.

Thanks!

**Comment by josenobile** [comment]
Hello, now that it has been merged, how long takes it to be available in wordpress.org? The deployment.

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.0.15

Feel free to update

Thanks!
